modules/configuration/src/main/java/org/apache/ignite/configuration/ConfigurationChanger.java

```diff
@@ -428,7 +428,7 @@ private void updateFromListener(
 
         ConfigurationStorage storage = storageInstances.get(storageType);
 
-        long storageRevision = changedEntries.storageRevision();
+        long storageRevision = changedEntries.cfgVersion();
```

Comment:
Can we rename it to "changeId" to avoid further confusion?

Reply:
done